proto/api/v2/report_service.proto

```diff
@@ -56,8 +59,11 @@ message VulnerabilityReportFilters {
     bool since_last_sent_scheduled_report = 5;
     google.protobuf.Timestamp since_start_date = 6;
   }
-  bool include_nvd_cvss = 7;
```

Comment:
Removing existing stuff could break backward compatibility. So might need to keep these around.